parlai/mturk/core/react_server/dev/components/core_components.jsx

```diff
@@ -312,11 +318,199 @@ class IdleResponse extends React.Component {
   }
 }
 
+class ReviewButtons extends React.Component {
+  GOOD_REASONS = [
+    'Not specified',
+    'Interesting/Creative',
+    'Other',
+  ]
+
+  BAD_REASONS = [
+    'Not specified',
+    "Didn't understand task",
+    "Bad grammar/spelling",
+    "Total nonsense",
+    "Slow responder",
+    'Other',
```

Comment:
nit: you switch between single and double quotes here (also using single quotes in `GOOD_REASONS`)